### PR TITLE
Remove selenium nyapp

### PR DESCRIPTION
**Pull request by flooie**

### CODE REVIEW

File: juriscraper/opinions/united_states/state/nyappterm_1st.py [hunk 1]
```diff
@@ -45,148 +31,32 @@ def __init__(self, *args, **kwargs):
...
+            self.cases.append(
+                {
+                    "name": row.xpath(".//td")[0].text_content(),
+                    "date": row.xpath(".//td")[1].text_content(),
+                    "url": url,
+                    "status": "Published"
+                    if "(U)" not in docket
+                    else "Unpublished",
+                    "docket": docket,
+                    "neutral_citation": docket,
```
Comment (mlissner) [inline juriscraper/opinions/united_states/state/nyappterm_1st.py:59]: They're doing something a bit weird here, I think. It seems like their docket numbers aren't provided in the search results page and you have to click through to the opinions to get it. Like, the first result I looked at was: https://www.nycourts.gov/reporter/3dseries/2021/2021_07058.htm

Which has Docket No. 79 and 78. 

Should we use our post process hook to get the docket numbers?

**Review by mlissner — COMMENTED** [review]
Bit of a biggie here.

File: juriscraper/opinions/united_states/state/nyappterm_1st.py (hunk 1 above)
Comment (flooie) [reply juriscraper/opinions/united_states/state/nyappterm_1st.py:59]: I think this was a failure of the previous scraper that I let slide.

Comment (mlissner) [reply juriscraper/opinions/united_states/state/nyappterm_1st.py:59]: Yeah, I figured as much. Now that the can of worms is open though, seems like we should fix it?

Comment (flooie) [reply juriscraper/opinions/united_states/state/nyappterm_1st.py:59]: indeed.  we should.

**Comment by flooie** [comment]
@mlissner back to you

File: juriscraper/opinions/united_states/state/nyappterm_1st.py (hunk 1 above)
Comment (flooie) [reply juriscraper/opinions/united_states/state/nyappterm_1st.py]: this should be done

**Comment by mlissner** [comment]
Awaiting response to comment here (sorry): https://github.com/freelawproject/juriscraper/commit/ec6573c9438702c09b7ad1888dc5b72f918f0170

**Comment by flooie** [comment]
@mlissner hm.... it says I pushed it... but I dont see it here.

**Comment by flooie** [comment]
Does [this](https://github.com/freelawproject/juriscraper/pull/461/commits/ec6573c9438702c09b7ad1888dc5b72f918f0170)
address it @mlissner

**Pull request closed without merging**